src/assets/styles/searchComponent.scss

```diff
@@ -0,0 +1,102 @@
+@import './variables.scss';
+
+.inp {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 2px solid #111f255e;
+  padding: 5px 15px;
+  outline: none;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+.navbar {
+  -webkit-border-radius: 0;
+  -moz-border-radius: 0;
+  border-radius: 0;
+}
+
+.no-content {
+  text-align: center;
+  margin-top: 200px;
+  font-size: 30px;
+}
+
+body {
+  margin-top: 20px;
+  background: #eee;
+}
+ul {
+  list-style-type: none;
+}
+
+.avatar {
+  width: 40px;
+  border-radius: 50%;
+}
+.btn {
+  display: inline-block;
+  padding: 6px 12px;
+  margin-bottom: 0;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.42857143;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: middle;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  background-image: none;
+  border: 1px solid transparent;
+  border-radius: 4px;
+}
+
+.wrapper {
+  position: relative;
+  width: 100%;
+  min-height: 50px;
+  margin: 0 auto;
+}
+.box {
```

Comment:
Kindly check this class out. It seems to override an existing class on the reset password pages. You could consider renaming it.

![bluereset](https://user-images.githubusercontent.com/33033576/59369073-479acd80-8d48-11e9-930a-a986e1d3fb5e.gif)